src/picker.js

```diff
@@ -104,13 +104,24 @@ function init({tabs: originalTabs, env}) {
 			}
 			const counter = document.createElement("div");
 			counter.className = "tab-image-counter";
+			counter.dataset.i18n = "pickerGroupCounterClicked|title";
+			counter.onclick = (e) => {
+				let imagesInGroup = getImages().filter(i => i.el.parentNode.lastElementChild === e.target);
+				let someUnchecked = imagesInGroup.some((i) =>
+					!i.el.classList.contains("disabled") && !i.el.classList.contains("checked"));
+				imagesInGroup.forEach(i => {
+					if (i.el.classList.contains("disabled")) return;
+					i.setCheck(e.shiftKey ? !someUnchecked : someUnchecked);
```

Comment:
I think there is no need for such complex logic involving shift key. Just a simple invert should be enough?

Reply:
It provides the Uncheck all action by default instead of Check all for a single group.
Suppose I have opened a capture for dozens of tabs, then selected a subset of images for multiple groups. But I decided to uncheck some selected images for the previous groups, but not to reload the capture to discard other status. Reversing all will affects subsequent groups.

In addition, an invert button in each group may be useful.

Reply:
Then I think we should have two buttons:
```html
<li class="image-container">
  ... labels ...
  <div class="tab-toolbar">
    <span class="image-counter"></span>
    <button class="toggle-all">Toggle all</button>
    <button class="invert">Invert</button>
  </div>
</li>
```

Reply:
I hope they're not too noisy. Buttons with text labels are wider. Only icon labels or a checkbox + "invert" (or rotating icon) button?